docs/templates/component.mdx

```diff
@@ -0,0 +1,37 @@
+---
+title: <component name>
+caption: <short purpose of component> (optional)
+figma: <link to figma> (optional)
+---
+
+<!-- longer description about capabilities and usage of the component -->
+
+```tsx onlyCode
+import {} from '@marigold/';
+```
+
+## Examples
```

Comment:
Examples is like showing different props in action or dont show every prop separate in an example?

Reply:
yes, see description in comment